python/robot_properties_solo/gepetto_gui_loader.py

```diff
@@ -38,7 +42,14 @@ def display_solo_in_gepetto_gui():
     config = SoloConfig()
     solo_visual.display(config.q0)
     # place the world frame
-    world_frame = Frame(gui_scene)    
+    world_frame = Frame(gui_scene)
+
+    if launch_gepetto_gui_exec:
+        # close the window after little while
+        time.sleep(5)
```

Comment:
Can you do better than doing a timer here? This slows down the entire setup and especially for learning is super annoying (where you launch such scripts a few thousand times).